sct-commons/src/main/java/org/lfenergy/compas/sct/commons/dto/SclReportItem.java

```diff
@@ -0,0 +1,33 @@
+// SPDX-FileCopyrightText: 2022 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.dto;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
```

Comment:
Why `ToString` ?

Reply:
Because when unit tests fail, you will have a nice display of the actual missing/not expected SclReportItem object.
Like in test `ExtRefServiceTest#updateAllExtRefIedNames_should_report_errors` for example which does a 
`assertThat(sclReport.getSclReportItems()).containsExactlyInAnyOrder(...)`
Otherwise when the test fails, the output is unusable.

Reply:
OK